behavior.c

```diff
@@ -397,7 +398,248 @@ static void pairing_tick(void) {
 		dbnc = 0;
 	}	
 	
-}	
+}
+
+static void setting_tick(void) {
```

Comment:
I am thinking whether the different behaviour should no go each into their own source file?